### PR TITLE
Make opening threads from a usercard opened with /usercard not crash the client

### LINKED ISSUE #3902: Clicking or prompting a reply from within a usercard causes a full crash
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

To reproduce:
- Send a message
- Right click -> Reply to Message
- `/usercard username`
- Click the reply

OR

- Send a message
- `/usercard username`
- Right click the message -> Reply to Message

Apparently from what @Mm2PL said, replies have split context, and not usercard context
I thought it would be good to make an issue to track 👍 

### Screenshots

Method 1:
https://i.alienpls.org/XrK.apng
Method 2:
https://i.alienpls.org/yhz.apng

### OS and Chatterino Version

Chatterino Nightly 2.3.5 (commit ebc7852f modified) built on 2022-07-31 with Qt 5.15.2, Windows SDK, MSVC 193231332 Running on Windows 10 Version 2004, kernel: 10.0.20152

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

It's a little hacky IMO but it works. Closes #3902.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,7 @@
 - Bugfix: Fix split focusing being broken in certain circumstances when the "Show input when it's empty" setting was disabled. (#3838, #3860)
 - Bugfix: Always refresh tab when a contained split's channel is set. (#3849)
 - Bugfix: Drop trailing whitespace from Twitch system messages. (#3888)
+- Bugfix: Fix crash caused by opening a reply thread from a usercard opened with /usercard (#3905)
```
Comment (dnsge) [inline CHANGELOG.md:49]: nit: Should this PR instead be included in the overall "Added support for Twitch's Chat Replies" changelog entry?

Comment (Mm2PL) [reply CHANGELOG.md:49]: Yeah, i'll move it

**Review by pajlada — APPROVED** [review]
I'm happy with this fix

There are some more ways you can crash things if you close the original split etc but that's more of a chatterino issue than this PR issue

**Review by Felanbird — APPROVED** [review]
it doesn't crash anymore

**Comment by Mm2PL** [comment]
Actually this still may crash if the split isn't in the main window

**Comment by pajlada** [comment]
Unable to crash this with popups

https://user-images.githubusercontent.com/962989/184422432-175be597-d7a1-4aa5-873b-c62c9d3bc5c0.mp4

**Pull request merged**